sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -32,6 +34,15 @@ export interface AnalyzeSentimentSuccessResult extends TextAnalyticsSuccessResul
     sentiment: DocumentSentimentLabel;
 }
 
+// @public (undocumented)
+export interface AspectSentiment {
+    confidenceScores: SentimentConfidenceScores;
+    // (undocumented)
+    sentiment: DocumentSentimentLabel;
+    // (undocumented)
+    text: string;
+}
```

Comment:
for arch board, we're trying to show the `offset` and `length` properties as well. Are you going to add them in a separate PR before arch board on Thursday? If not, I would suggest adding them now

Reply:
I will add them in another PR.